app/src/gui/components/record/relationships/field_level_links/datagrid.tsx

```diff
@@ -305,18 +340,25 @@ export function DataGridFieldLinksComponent(
     valueGetter: (params: GridCellParams) => params.row.updated_by,
   };
 
-  const columns: any =
-    props.relation_type === 'Child'
-      ? [record_column, updated_by_column]
-      : [relation_column, record_column, updated_by_column];
+  const columns: GridColDef[] = [];
+
+  if (props.relation_type !== 'Child') columns.push(relation_column); // keep relationship column
```

Comment:
Note that adding this relation column on small screen messes up the display - it should become part of the details column.